<launch>
	
    <node pkg="rr_endline_detection" type="endline_detection" name="endline_detection_node" output="screen" required="true">
    </node>
    
    
    <!-- <include file="$(find rr_endline_detection)/launch/webcam_test.launch" /> -->

</launch>